node_modules/col-activities/lib/notifications/weekly.js

```diff
@@ -154,19 +154,24 @@ var handleUser = function(course, activitySummary, user, callback) {
 /**
  * Get activity and user data for a given course in the past week.
  *
- * @param  {Context}       course                                                The course for which to get activity data
- * @param  {Function}      callback                                              Standard callback
- * @param  {Object}        callback.err                                          An error object, if any
- * @param  {Object}        callback.courseData                                   Course data for the weekly email
- * @param  {Object}        callback.courseData.activitySummary                   Summary of the past week's activity in the course
- * @param  {Object}        callback.courseData.activitySummary.assets            Assets with activity this week, indexed by id
- * @param  {Object}        callback.courseData.activitySummary.course            Course-wide weekly data
- * @param  {Object}        callback.courseData.activitySummary.course.averages   Course-wide activity and point averages for the week
- * @param  {Object}        callback.courseData.activitySummary.course.topAssets  Highest-scoring assets for the week
- * @param  {Object}        callback.courseData.activitySummary.course.topUsers   Highest-scoring users for the week
- * @param  {CourseTotals}  callback.courseData.activitySummary.course.totals     Course-wide activity and point totals for the week
- * @param  {Object}        callback.courseData.activitySummary.users             User activity and point totals, indexed by id
- * @param  {User[]}        callback.data.users                                   All active users in the course
+ * @param  {Context}       course                                                               The course for which to get activity data
+ * @param  {Function}      callback                                                             Standard callback
+ * @param  {Object}        callback.err                                                         An error object, if any
+ * @param  {Object}        callback.courseData                                                  Course data for the weekly email
+ * @param  {Object}        callback.courseData.activitySummary                                  Summary of the past week's activity in the course
+ * @param  {Object}        callback.courseData.activitySummary.assets                           Assets with activity this week, indexed by id
+ * @param  {Object}        callback.courseData.activitySummary.course                           Course-wide weekly data
+ * @param  {Object}        callback.courseData.activitySummary.course.averages                  Course-wide activity and point averages for the week
+ * @param  {Object}        callback.courseData.activitySummary.course.topAssets                 Highest-scoring assets for the week
+ * @param  {Object}        callback.courseData.activitySummary.course.topAssets.comments        The asset with the most comments for the week
```

Comment:
That's a hard one. So far, we've used `Object` and have specified in the description that it's an object containing assets keyed by asset id.